src/lib/components/data-graphic/guides/Axis.svelte

```diff
@@ -123,22 +130,40 @@ This component will draw an axis on the specified side.
     const fourDaysDiff = diff / (60 * 60) < 24 * 4;
     const manyDaysDiff = diff / (60 * 60 * 24) < 60;
     const manyMonthsDiff = diff / (60 * 60 * 24) < 365;
-    return millisecondDiff
-      ? timeFormat("%M:%S.%L")
-      : secondDiff
-      ? timeFormat("%M:%S")
-      : dayDiff
-      ? timeFormat("%H:%M")
-      : fourDaysDiff || manyDaysDiff || manyMonthsDiff
-      ? timeFormat("%b %d")
-      : timeFormat("%Y");
+
+    if (millisecondDiff) {
```

Comment:
Could we have each of these cases return an array of superlabelFormatter, mainlabelFormatter, and then apply those? That'll make it easier to test when we refactor this and clean up the axis